kalite/contentload/tests/unpack_assessment_zip.py

```diff
@@ -10,21 +11,47 @@
 
 from kalite.testing import KALiteTestCase
 from kalite.contentload.management.commands import unpack_assessment_zip as mod
-
-ASSESSMENT_ZIP_SAMPLE_PATH = os.path.join(
-    os.path.dirname(__file__),
-    "fixtures",
-    "assessment_item_resources.sample.zip"
-)
+from kalite import version
 
 
 class UnpackAssessmentZipCommandTests(KALiteTestCase):
 
+    def setUp(self):
+        _, self.zipfile_path = tempfile.mkstemp()
+        with open(self.zipfile_path, "w") as f:
+            zf = zipfile.ZipFile(f, "w")
+            for dirpath, _, filenames in os.walk(os.path.join(os.path.dirname(__file__), "fixtures")):
+                # this toplevel for loop should only do one loop, but
+                # it's in a for loop nonetheless since it's more idiomatic
+                for filename in filenames:
+                    full_path = os.path.join(dirpath, filename)
+                    zf.write(full_path, filename)
```

Comment:
For these tests, I used to have a zipfile committed in git inside `fixtures/`. Now I just generate this zip on the fly using the contents found in `fixtures/`.